.github/workflows/pr-check.yml

```
@@ -0,0 +1,343 @@
+          api-level: 33
+          build-tools: '33.0.2'
+
+      # Create local.properties
+      - name: Create local.properties
+        run: |
+          echo "sdk.dir=$ANDROID_HOME" > local.properties
+          echo "GOOGLE_KEY=${{ secrets.GOOGLE_MAPS_API }}" >> local.properties
+
+      # Create release.secrets.properties
+      - name: Create release.secrets.properties
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)